File: Examen/C#/Examen-1.cs
using System;

public class SumadoraSimple
{
    public static void Main(string[] args)
    {
        Console.WriteLine("Por favor, Introduzca un numero: ");
        string entrada1 = Console.ReadLine();

        Console.WriteLine("Por favor, Introduzca un numero: ");
        string entrada2 = Console.ReadLine();

        try
        {
            int num1 = Convert.ToInt32(entrada1);
            int num2 = Convert.ToInt32(entrada2);

            int suma = num1 + num2;

            Console.WriteLine($"La suma de {num1} y {num2} es: {suma}");
        }
        catch (FormatException)
        {
            Console.WriteLine("Error: Por favor, introduce solo números válidos.");
        }
    }
}

File: Examen/JavaScript/Examen-1.js
function calculadora(num1, num2, operador){
    switch(operador){
        case '+':
            return  num1 + num2;
            
        case '-':
            return  num1 - num2;

        case '*':
            return num1 * num2;

        case '/':

            if(num2 === 0){
                return "Error: Division por Cero"
            }

            default:
                return num1 / num2;
            
    }   

}

console.log(`2 + 3 = ${calculadora(2, 3, '+')}`);   
console.log(`10 - 5 = ${calculadora(10, 5, '-')}`);  
console.log(`4 * 6 = ${calculadora(4, 6, '*')}`);    
console.log(`10 / 2 = ${calculadora(10, 2, '/')}`);  
console.log(`5 / 0 = ${calculadora(5, 0, '/')}`); 